pipettes/firmware/main.cpp

```diff
@@ -118,6 +119,24 @@ static sensors::hardware::SensorHardware pins_for_sensor_96(gpio::PinConfig{
     .pin = GPIO_PIN_5,
     .active_setting = GPIO_PIN_RESET});
 
+// Unfortunately, these numbers need to be literals or defines
+// to get the compile-time checks to work so we can't actually
+// correctly rely on the hal to get these numbers - they need
+// to be checked against current configuration. However, they are
+// - clock input is 110MHz assuming the CAN is clocked from sysclk
+// - 455ns requested time quantum yields a 454ns actual
+// - 275.33KHz bitrate
+// - 87.5% sample point
+// Should drive
+// segment 1 = 8 quanta
+// segment 2 = 1 quantum
+
+// For the exact timing values these generate see
+// can/tests/test_bit_timings.cpp
+static constexpr auto can_bit_timings =
+    can::bit_timings::BitTimings<110 * can::bit_timings::MHZ, 455, 275330,
```

Comment:
275.33 * can::bit_timings::KHZ ?

Reply:
Resolved by [follow-up](https://github.com/Opentrons/ot3-firmware/pull/349/files), so you can ignore my comment.